app/models/category.rb

```diff
@@ -1,6 +1,10 @@
 class Category < ApplicationRecord
+  extend FriendlyId
+  friendly_id :name, use: :slugged
+
   has_and_belongs_to_many :posts
 
   validates :name, presence: true, length: { maximum: 255 }, uniqueness: true
   validates :description, length: { maximum: 1000 }
+  validates :slug, presence: true
```

Comment:
isn't friendly_id already taking care of that?

Reply:
In their guide, they also validate the presence of the slug, see http://norman.github.io/friendly_id/file.Guide.html#Example_Setup

Not sure how up to date these guides are, though.

Reply:
ok, thx. 
und schadet bestimmt eh nicht. :)